### DISCUSSION TITLE
Empty cards on the feed

### DISCUSSION THREAD

**Discussion by llrs** [community/community#13185] [Other Feature Feedback, Questions, & Ideas]
Perhaps if a user creates a repository without any information or README there is no need to show a card (this is on the following tab):

![Screenshot from 2022-03-21 23-46-30](https://user-images.githubusercontent.com/6818218/159375440-cf3154ff-734c-4334-8879-7d9ab71a599d.png)
Or it could show once it has a README and some more information about the repository

**Comment by jfullstackdev** [comment]
is this fixed by now ? 

just to update also

there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by llrs** [reply]
I don't see any empty repo on the feed. But I don't know if this is due to a fix or because no account I follow has an empty repository.
